README.md

```diff
@@ -19,6 +19,18 @@ text.
 To contribute, fork this repo, and submit a PR. We'll review, discuss, and merge
 changes as needed. Thank you!
 
-# To Do List:
+# Translations available
 
-We'd like to expand the current CoC to include sections for our community channels including forums and IRC, and our meetups.
+The Code of Conduct is currently available in:
+- English [code-of-conduct](code-of-conduct.md)
+- Portuguese (Brazilian) [code-of-conduct-PT](code-of-conduct-PT.md) (contributors: @fike, @joaofnfernandes)
+
+# Wishlist/To Do List:
+
+We welcome contributions from community members interested in helping us maintain, and expand the coverage of this CoC.  To that end, we have a wishlist for edits:
+
+- include a section for community channels
+  - forums
+  - Slack and/or IRC
+  - Community meetups
```

Comment:
Community events including meetups is covered in the existing content. 
+1 on adding verbiage around CoC specific to online forums and exchanges